fft/src/ops.rs

```diff
@@ -7,22 +7,28 @@ use crate::roots_of_unity::get_twiddles;
 
 use super::{
     bit_reversing::in_place_bit_reverse_permute, errors::FFTError,
-    fft_iterative::in_place_nr_2radix_fft, helpers::log2,
+    fft_iterative::in_place_nr_2radix_fft,
 };
 
+use super::helpers;
+
 /// Executes Fast Fourier Transform over elements of a two-adic finite field `F` in a coset. Usually used for
 /// fast polynomial evaluation.
 pub fn fft_with_blowup<F: IsTwoAdicField>(
-    coeffs: &[FieldElement<F>],
+    input: &[FieldElement<F>],
```

Comment:
Hmm, what was the thought process behind the rename?

Reply:
I think these functions were first made specifically for polynomial evaluation, and later the API got generalized for any slice of field elements, but these names remained referencing the coefficients of a polynomial. So `input` is a better name (and it matches other `ops.rs` files). This isn't really related to the issue but it was annoying me lol